code/game/objects/items/weapons/twohanded.dm

```diff
@@ -511,4 +513,60 @@
 	return 0
 
 /obj/item/weapon/twohanded/vibro_weapon/update_icon()
-	icon_state = "hfrequency[wielded]"
\ No newline at end of file
+	icon_state = "hfrequency[wielded]"
+
+/*
+ * Bone Axe
+ */
+/obj/item/weapon/twohanded/fireaxe/boneaxe  // Blatant imitation of the fireaxe, but made out of bone.
+	icon_state = "bone_axe0"
+	name = "bone axe"
+	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/update_icon()  //Currently only here to fuck with the on-mob icons.
+	icon_state = "bone_axe[wielded]"
+	return
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] axes \himself from head to toe! It looks like \he's trying to commit suicide..</span>")
+	return (BRUTELOSS)
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/afterattack(atom/A as mob|obj|turf|area, mob/user, proximity)
```

Comment:
uh if this is the same as normal axes you don't need to add this, it's inherited